tests/odm/documents/test_bulk_write.py

```diff
@@ -22,6 +22,7 @@ async def test_insert(documents_not_inserted):
 
 async def test_update(documents, document_not_inserted):
     await documents(5)
+    DocumentTestModel.invalidate_cache()
```

Comment:
as this is not a part of the test, it is better to move it as a fixture to the modular conftest file and run it in `autouse=True` mode